codecs/codecs-btp/pom.xml

```diff
@@ -9,46 +9,30 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <name>Interledger Core Codecs</name>
-  <description>Codecs for the Interledger Core classes.</description>
-  <artifactId>ilp-core-codecs</artifactId>
+  <name>Bilateral Transfer Protocol Core Codecs</name>
+  <url>http://maven.apache.org</url>
+  <description>BTP Protocol Core Codecs.</description>
+  <artifactId>codecs-btp</artifactId>
 
   <dependencies>
-    <dependency>
-      <groupId>${project.groupId}</groupId>
-      <artifactId>codec-framework</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>${project.groupId}</groupId>
-      <artifactId>ilp-core</artifactId>
-    </dependency>
-
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
-
-    <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
-      <scope>test</scope>
-    </dependency>
-
     <dependency>
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest-all</artifactId>
       <scope>test</scope>
     </dependency>
-
     <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-      <scope>test</scope>
+      <groupId>org.interledger</groupId>
+      <artifactId>btp-core</artifactId>
     </dependency>
-
   </dependencies>
 
+  <properties>
```

Comment:
This should be necessary - it should be defined in the parent pom and inherited.